trunk/src/app/srs_app_rtc_source.cpp

```diff
@@ -2666,6 +2674,24 @@ SrsRtpPacket* SrsRtcSendTrack::fetch_rtp_packet(uint16_t seq)
     // Return packet only when sequence is equal.
     if (pkt->header.get_sequence() == seq) {
         ++_srs_pps_rhnack->sugar;
+
+        if ((pkt->resend_ms_ <= 0) || (pkt->resend_count_ == 0)) {
+            pkt->resend_ms_ = now_ms;
+            pkt->resend_count_++;
+        } else {
+            int64_t diff_t = now_ms - pkt->resend_ms_;
+            int64_t interval = (int64_t)avg_rtt_;
+            interval = (interval > 10) ? (interval - 10) : interval;//for resend interval Residual
+            if (diff_t < (int64_t)interval) {
+                return NULL;
+            }
```

Comment:
Should limit the count of retrans packet?

Reply:
done